module/VuFind/tests/unit-tests/src/VuFindTest/OAuth2/Repository/AbstractTokenRepositoryTestCase.php

```diff
@@ -355,24 +313,42 @@ protected function getMockAccessTokenService(): AccessTokenServiceInterface
         return $accessTokenService;
     }
 
+    /**
+     * Create User entity mock.
+     *
+     * @param int    $id       User ID
+     * @param string $username User's name
+     *
+     * @return MockObject&UserEntityInterface
+     */
+    protected function createMockUserEntity(int $id, string $username): UserEntityInterface|MockObject
+    {
+        $mockUser = $this->createMock(UserEntityInterface::class);
+        $mockUser->expects($this->any())
+        ->method('getId')
+        ->willReturn($id);
+        $mockUser->expects($this->any())
+        ->method('getUsername')
+        ->willReturn($username);
```

Comment:
Some suggested indentation adjustments:

```suggestion
        $mockUser->expects($this->any())
            ->method('getId')
            ->willReturn($id);
        $mockUser->expects($this->any())
            ->method('getUsername')
            ->willReturn($username);
```